packages/edge-login-ui-rn/src/actions/LoginInitActions.js

```diff
@@ -109,9 +109,7 @@ const checkAndRequestNotifications = () => async (
   getState: GetState,
   imports: Imports
 ) => {
-  const notificationPermision = await checkNotifications().catch(error =>
-    console.log(error)
-  )
+  const notificationPermision = await checkNotifications()
```

Comment:
Are you sure you don't need to catch here?

Reply:
The catch is a bug. If you catch here, you will crash on the next line, because `notificationPermision` will be undefined. Instead, let the throw exit the whole routine, and then the outer code will catch the exception.